### PR TITLE
Deleted regions filter on navbar

### PR DESCRIPTION
**Pull request by paurisa**
Deleted the **Regions** filters on the navbar as specified in the first item of issue #647 

Also added the name of the cloud provider to the CLI output and fixed some typos.

### CODE REVIEW

**Comment by Aboisier** [comment]
# [Codecov](https://codecov.io/gh/nccgroup/ScoutSuite/pull/652?src=pr&el=h1) Report
> Merging [#652](https://codecov.io/gh/nccgroup/ScoutSuite/pull/652?src=pr&el=desc) into [develop](https://codecov.io/gh/nccgroup/ScoutSuite/commit/9a2538769abc435ad2932d0e42c96d3b898f0212&el=desc) will **not change** coverage by `%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/nccgroup/ScoutSuite/pull/652?src=pr&el=tree)

```diff
@@           Coverage Diff            @@
##           develop     #652   +/-   ##
========================================
  Coverage    65.21%   65.21%           
========================================
  Files           22       22           
  Lines         1515     1515           
========================================
  Hits           988      988           
  Misses         527      527           
```


| [Impacted Files](https://codecov.io/gh/nccgroup/ScoutSuite/pull/652?src=pr&el=tree) | Coverage Δ | |
|---|---|---|
| [ScoutSuite/output/result\_encoder.py](https://codecov.io/gh/nccgroup/ScoutSuite/pull/652/diff?src=pr&el=tree#diff-U2NvdXRTdWl0ZS9vdXRwdXQvcmVzdWx0X2VuY29kZXIucHk=) | `31.63% <0.00%> (ø)` | :arrow_up: |
| [ScoutSuite/providers/base/resources/base.py](https://codecov.io/gh/nccgroup/ScoutSuite/pull/652/diff?src=pr&el=tree#diff-U2NvdXRTdWl0ZS9wcm92aWRlcnMvYmFzZS9yZXNvdXJjZXMvYmFzZS5weQ==) | `70.00% <0.00%> (ø)` | :arrow_up: |
| [ScoutSuite/core/server.py](https://codecov.io/gh/nccgroup/ScoutSuite/pull/652/diff?src=pr&el=tree#diff-U2NvdXRTdWl0ZS9jb3JlL3NlcnZlci5weQ==) | `31.64% <0.00%> (ø)` | :arrow_up: |
| [ScoutSuite/core/processingengine.py](https://codecov.io/gh/nccgroup/ScoutSuite/pull/652/diff?src=pr&el=tree#diff-U2NvdXRTdWl0ZS9jb3JlL3Byb2Nlc3NpbmdlbmdpbmUucHk=) | `82.35% <0.00%> (ø)` | :arrow_up: |
| [ScoutSuite/core/rule.py](https://codecov.io/gh/nccgroup/ScoutSuite/pull/652/diff?src=pr&el=tree#diff-U2NvdXRTdWl0ZS9jb3JlL3J1bGUucHk=) | `95.78% <0.00%> (ø)` | :arrow_up: |

------

[Continue to review full report at Codecov](https://codecov.io/gh/nccgroup/ScoutSuite/pull/652?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/nccgroup/ScoutSuite/pull/652?src=pr&el=footer). Last update [9a25387...8a08c04](https://codecov.io/gh/nccgroup/ScoutSuite/pull/652?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by j4v** [comment]
@paurisa great, I'll review today :+1:

File: ScoutSuite/__main__.py [hunk 1]
```diff
@@ -177,7 +177,7 @@ def run(provider,
 
     print_info('Launching Scout')
 
-    print_info('Authenticating to cloud provider')
+    print_info('Authenticating to %s cloud provider' % provider.upper())
```
Comment (j4v) [inline ScoutSuite/__main__.py]: Ideally we'd format the name, but that's only available once the `cloud_provider` object is instantiated.

Comment (paurisa) [reply ScoutSuite/__main__.py]: I just added this minor change to make it easier when calling ScoutSuite against a cloud provider or if the output of the tool is later used somehow.
You can discard this change if you find it's not necessary.

**Comment by j4v** [comment]
@paurisa looks :100:, please merge!

**Pull request merged**